Source/VisualStyle.swift

```diff
@@ -105,7 +110,7 @@ extension VisualStyle {
 public class DefaultVisualStyle: VisualStyle {
 
     private let alertStyle: AlertControllerStyle
-    init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
+    public init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
```

Comment:
Ahh I see, that makes sense. Out of curiosity, how did you init your subclasses before this change then? Did you have to add your own initializers?